### DISCUSSION TITLE
SvgImage source is not dependency property so bindings are failed

### DISCUSSION THREAD

**Discussion by eladpotok** [ElinamLLC/SharpVectors#212] [Q&A]
Hi,
Trying to bind between a viewmodel image url to the SvgImage.Source but it is failed since Source is not declared as dependency property.

Can you please change it? or maybe there is another way ?

thanks

**Comment by paulushub** [comment]
@eladpotok `SvgImage` is a markup extension, and cannot support dependency property and  binding (WPF limitations).
`SvgImageConverter` is a similar markup extension, but implements `IValueConverter` to support binding.
See the samples in the **/TutorialSamples/ControlSamplesWpf** directory to see how to use these features and binding samples.
